frontend/lib/src/components/widgets/NumberInput/NumberInput.test.tsx

```diff
@@ -414,12 +363,11 @@ describe("NumberInput widget", () => {
         default: 10,
         step: 1,
       })
-      const wrapper = shallow(<NumberInput {...props} />)
+      render(<NumberInput {...props} />)
 
-      stepDownButton(wrapper).simulate("click")
-
-      expect(wrapper.state("dirty")).toBe(false)
-      expect(wrapper.state("value")).toBe(9)
+      // Increment
+      fireEvent.click(screen.getByTestId("stNumberInput-StepDown"))
+      expect(screen.getByTestId("stNumberInput-Input")).toHaveValue(9)
     })
 
     it("handles stepUp button clicks", () => {
```

Comment:
Isn't this the same as the test above (`it("changes state on ArrowUp", () =>`)? Can't we just remove one of them or combine the titles or something? will leave it up to you

Reply:
Not entirely, one is clicking the input field then arrowUp on the keyboard, other is using the actual stepUp (+) button next to the input field